ui/core/src/clients/bridges/IBCBridge/IBCBridge.ts

```diff
@@ -264,13 +264,10 @@ export class IBCBridge extends BaseBridge<CosmosWalletProvider> {
       throw new Error("Invalid transfer symbol not in whitelist: " + symbol);
     }
 
-    const totalSupply = await (
-      await provider.getQueryClient(params.fromChain)
-    ).bank.totalSupply();
-
     let transferDenom: string;
     if (
-      totalSupply.some((coin) => coin.denom === transferTokenEntry.baseDenom)
```

Comment:
This line here was causing issues; an ibc denom was being picked for `uatom` import. Not sure when it was added, git blame is polluted by a giant file reorg

Reply:
hmm. might break when we implement multihop IBC but otherwise looks good.